examples/next-forms/app/actions.ts

```diff
@@ -24,25 +24,29 @@ export async function createTodo(formData: FormData) {
   `
 
     revalidatePath('/')
-
-    return { message: 'Saved successfully' }
+    return { message: `Added todo ${data.todo}` }
   } catch (e) {
     return { message: 'Failed to create todo' }
   }
 }
 
-export async function deleteTodo(formData: FormData) {
+export async function deleteTodo(prevState: any, formData: FormData) {
   const schema = z.object({
     id: z.string().nonempty(),
   })
   const data = schema.parse({
     id: formData.get('id'),
   })
 
-  await sql`
-    DELETE FROM todos
-    WHERE id = ${data.id};
-  `
+  try {
+    await sql`
+      DELETE FROM todos
+      WHERE id = ${data.id};
+    `
 
-  revalidatePath('/')
+    revalidatePath('/')
+    return { message: 'Deleted todo' }
```

Comment:
```suggestion
    return { message: `Deleted todo ${data.todo}` }
```

Same change as for adding, we need to make sure the announcement text is different between deletions so the AT user will hear the feedback.